iot_config/src/org.rs

```diff
@@ -1,478 +1,167 @@
-use crate::{
-    helium_netids::{self, is_helium_netid, AddressStore, HeliumNetId},
-    lora_field::{DevAddrConstraint, DevAddrField, NetIdField},
-    org_service::UpdateAuthorizer,
-};
+use crate::lora_field::{DevAddrConstraint, NetIdField};
 use futures::stream::StreamExt;
-use helium_crypto::{PublicKey, PublicKeyBinary};
+use rust_decimal::{prelude::ToPrimitive, Decimal};
 use serde::Serialize;
-use sqlx::{postgres::PgRow, types::Uuid, FromRow, Row};
+use solana_sdk::pubkey::Pubkey;
+use sqlx::{error::Error as SqlxError, postgres::PgRow, types::Uuid, FromRow, Row};
 use std::collections::HashSet;
+use std::str::FromStr;
 use tokio::sync::watch;
 
 pub mod proto {
-    pub use helium_proto::services::iot_config::{
-        org_update_req_v1::update_v1::Update, org_update_req_v1::UpdateV1, ActionV1, OrgResV1,
-        OrgV1,
-    };
+    pub use helium_proto::services::iot_config::{ActionV1, OrgResV2, OrgV2};
 }
 
 #[derive(Clone, Debug, Serialize)]
 pub struct Org {
     pub oui: u64,
-    pub owner: PublicKeyBinary,
-    pub payer: PublicKeyBinary,
+    pub address: Pubkey,
+    pub owner: Pubkey,
+    pub escrow_key: String,
+    pub approved: bool,
     pub locked: bool,
-    pub delegate_keys: Option<Vec<PublicKeyBinary>>,
+    pub delegate_keys: Option<Vec<Pubkey>>,
     pub constraints: Option<Vec<DevAddrConstraint>>,
 }
 
 impl FromRow<'_, PgRow> for Org {
     fn from_row(row: &PgRow) -> sqlx::Result<Self> {
-        let delegate_keys = row
-            .get::<Vec<PublicKeyBinary>, &str>("delegate_keys")
-            .into_iter()
-            .map(Some)
-            .collect();
-        let constraints = row
-            .get::<Vec<(i32, i32)>, &str>("constraints")
-            .into_iter()
-            .map(|(start, end)| {
-                Some(DevAddrConstraint {
-                    start_addr: start.into(),
-                    end_addr: end.into(),
+        let address_str: String = row.get("address");
+        let address = Pubkey::from_str(&address_str).map_err(|e| SqlxError::Decode(Box::new(e)))?;
```

Comment:
does pubkey support `parse()` to combine these lines? `row.get("address").parse().map_err(etc)`